IndoorMapsBackend/src/__tests__/buildings.test.ts

```diff
@@ -51,8 +51,8 @@ describe('Testing the GraphQL server by running a HttpServer', () => {
         variables: {
             "data": {
                 "password": "pass",
-                "name": testBuildingName+"creator",
-                "email": testBuildingName+"creator" + "@test.com"
+                "name": testBuildingName + "creator",
+                "email": testBuildingName + "creator" + "@test.com"
```

Comment:
nit another cleaner way to write string concatenation by adding backticks ('):
"email": `${testBuildingName}creator@test.com`